test/controllers/organizations/home_controller_test.rb

```diff
@@ -3,6 +3,11 @@
 class Organizations::HomeControllerTest < ActionDispatch::IntegrationTest
```

Comment:
For all of these tests, I gather you had to create a PageText for each org so there are no errors when the view calls for the org's page text.

Instead of creating them in the setup of each test class, can't we just create the PageText when we create an organization in the factories? Then any time we create an org via factory, which is pretty much most tests, there will be an associated PageText, and there won't be any need to have to add this to every test setup. 

For example, check out the factory for organizations, and you will see that we pass a `profile` object to the Organization. We could also pass it a PageText object in the same way, so then there is no need to create one after the fact.

Does that make sense? Then we should be able to get rid of all these additions to the test setups.